### PR TITLE
Add first description of NEST

### PR DESCRIPTION
**Pull request by terhorstd**
This PR adds the project page for the NEST Simulator.

### CODE REVIEW

File: _projects/nest.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Members: https://github.com/orgs/nest/people and https://nest-initiative.org
+Link: https://nest-simulator.org
+Research_Field: Information
+Scientific_Community: Biology, Medicine, AI
+Funding: See https://github.com/nest/nest-simulator/blob/master/ACKNOWLEDGMENTS.md
+Programming_Languages: C++, Python
+License: GPL-2.0-or-later
+Cite:  <a href="https://doi.org/10.5281/zenodo.5886894" alt="NEST 3.2"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.5886894.svg" alt="DOI"></a>
+toc: true
+---
```
Comment (pancetta) [inline _projects/nest.md:16]: @steffengraber Can we have all these items in the current form?

File: _projects/nest.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+<!-- background, community and friends -->
+The simulator is developed and continuously improved by the NEST community around the globe to stay up-to-date and fit for the latest research questions. NEST developers are using continuous-integration based workflows in order to maintain high code quality standards for correct and reproducible simulations.
+
+<!-- Community building -->
+NEST has fostered a large community of experienced developers and amazing users, who actively contribute to the project. Our community extends to related projects, like the teaching tool [NEST Desktop](https://nest-desktop.readthedocs.io/en/latest/), cross-simulator languages like [PyNN](https://neuralensemble.org/docs/PyNN/) and neural activity analysis tools like [Elephant](https://elephant.readthedocs.io/en/latest/).
+
+With the combination of AI and neuromorphic computing, both sides can advance from findings from both worlds. Connect your NEST simulation to [neurorobotics](https://neurorobotics.net/) and add the "eyes" and "muscles" to complete the sensor-motor loop and tackle the problem of embodiment.
+
```
Comment (pancetta) [inline _projects/nest.md:47]: Goals missing, yes, yes, I know.

File: _projects/nest.md (hunk 1 above)
Comment (steffengraber) [reply _projects/nest.md:16]: Yes. Do we want to allow any entry in the future or do we give a list of possible entries.

Comment (pancetta) [reply _projects/nest.md:16]: Any entry is good, who knows what people want.. and we check the PRs anyway.

**Pull request merged**